microscopium/features.py

```diff
@@ -265,3 +287,43 @@ def nuclei_per_cell_histogram(nuc_im, cell_im, max_value=10):
     fs /= total
     return fs, names
 
+
+@tz.curry
+def default_feature_map(image, channels=[0, 1, 2], channel_names=None,
```

Comment:
Is there's any use in calling the curry function here? Every parameter except ``image`` already has a default? Or am I missing something here? :P